operator_toolkit/soc_helpers/config.yml

```
@@ -0,0 +1,13 @@
+influxdb_client:
+  epoch: ns
+  ssl: True
+  chunked: False
+  dropna: False
+  host: greatscott-b8391c4a.influxcloud.net
+  port: 8086
+  database: axiom-cloud
+  interval: 1m
+  timeout: 900
+  timezone: UTC
+query_fields:
+  - rack.*.SoC_.*_percentile$
```

The more explicit regex would look like:
```suggestion
  - rack\d+\.sg\d+\.SoC_.*_percentile$
```